src/machine learning analysis.py

```diff
@@ -0,0 +1,170 @@
+# author: Siqi Tao
+# date: 2021-11-26
+
+"""Downloads data as a csv file from online to a local filepath via a URL.
+Usage: download_script.py --url=<url> --filepath=<filepath> 
+ 
+Options:
+--input=<input>       The path or filename pointing to the data
+--prefix=<prefix>     The prefix where to write the output figure(s)/table(s) to and what to call it 
```

Comment:
Need to standardize this between scripts. It will be confusing if the commands keep changing. The way I wrote it was just --output